Clang/MacOS compiler error in opc-ua com layer

Clang on MacOS (via 4diac-fbe develop branch) fails in the OPC-UA layer:

```
In file included from /home/jwalter/lcl/4diac-fbe/build/test-macos/forte/build/com/opc_ua/opcua_types.cpp:12:
/home/jwalter/lcl/4diac-fbe/forte/src/com/opc_ua/opcua_helper.h:86:40: error: implicit instantiation of undefined template 'std::vector<const UA_Variant *>'
   86 |         std::vector<const UA_Variant*> mData;
      |                                        ^
/home/jwalter/lcl/4diac-fbe/toolchains/clang-toolchain/SDK/MacOSX11.1.sdk/usr/include/c++/v1/iosfwd:216:28: note: template is declared here
  216 | class _LIBCPP_TEMPLATE_VIS vector;
      |                            ^
In file included from /home/jwalter/lcl/4diac-fbe/build/test-macos/forte/build/com/opc_ua/opcua_types.cpp:12:
/home/jwalter/lcl/4diac-fbe/forte/src/com/opc_ua/opcua_helper.h:100:34: error: implicit instantiation of undefined template 'std::vector<UA_Variant *>'
  100 |         std::vector<UA_Variant*> mData;
      |                                  ^
/home/jwalter/lcl/4diac-fbe/toolchains/clang-toolchain/SDK/MacOSX11.1.sdk/usr/include/c++/v1/iosfwd:216:28: note: template is declared here
  216 | class _LIBCPP_TEMPLATE_VIS vector;
```

--- a/src/com/opc_ua/opcua_helper.h
+++ b/src/com/opc_ua/opcua_helper.h
@@ -20,6 +20,7 @@
 #include "forte_any.h"
 
 #include <string>
+#include <vector>
 
 class CIEC_STRUCT;
 

